src/panel/context/Events.tsx

```diff
@@ -149,6 +136,11 @@ export const EventsProvider: FC = ({ children }) => {
   return <EventsContext.Provider value={value} children={children} />;
 };
 
+export const genEventsFromRaw = (rawEvents: PresentedEvent[]) =>
```

Comment:
~We shouldn't need to touch this - context is laid out in a way which means each panel has it's own state [see here](https://github.com/FormidableLabs/urql-devtools/blob/master/src/panel/App.tsx#L35)~

Looks like this is just refactoring so nothing to see here :+1:

Reply:
The filter should still be active [here](https://github.com/FormidableLabs/urql-devtools/blob/7f956bb1d8c4565d628e37bdf85b3e8c88812943/src/panel/context/Events.tsx#L122)

Happy to revert the changes here though as they're not really timeline-related